src/System.Private.Xml/tests/XmlReader/XmlResolver/XmlSystemPathResolverTests.cs

```diff
@@ -29,6 +32,7 @@ public static void TestResolveRelativePaths()
                 if (shouldDelete)
                 {
                     File.Delete(path);
+                    Directory.SetCurrentDirectory(curDir);
```

Comment:
I meant to change that in my last commit, but forgot. Changed that now. I believe it's there only to prevent populating the current directory?